superbench/benchmarks/micro_benchmarks/cublas_function/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.18)
+project(CublasBenchmark)
+
+include(../cuda_common.cmake)
+
+SET(SRC "cublas_benchmark.cu" CACHE STRING "source file")
+SET(TARGET_NAME "cublas_function" CACHE STRING "target name")
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+set(CMAKE_CXX_FLAGS "-Wall -Wextra -std=c++11 -march=native")
+set(CMAKE_CXX_FLAGS_DEBUG "-g")
+set(CMAKE_CXX_FLAGS_RELEASE "-O2")
+
+find_package(CUDA)
+set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} ${NVCC_ARCHS_SUPPORTED}")
+# set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}  -ftemplate-depth=4096 -gencode arch=compute_60,code=sm_60 -gencode arch=compute_61,code=sm_61 -gencode arch=compute_70,code=sm_70 -gencode arch=compute_75,code=sm_75")
+set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -O2")
+set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -cudart shared")
+set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} --expt-relaxed-constexpr")
+cuda_add_library(${TARGET_NAME} SHARED ${SRC})
+
+link_directories(${CUDA_TOOLKIT_ROOT_DIR}/lib64)
+
+find_path(CUDNN_INCLUDE_DIR cudnn.h
+    HINTS ${CUDA_TOOLKIT_ROOT_DIR}
+    PATH_SUFFIXES cuda/include include)
+
+include_directories(${CUDNN_INCLUDE_DIR} ${CUDA_INCLUDE_DIRS})
+
+find_library(CUDNN_LIBRARY cudnn
+    HINTS ${CUDA_TOOLKIT_ROOT_DIR}
+    PATH_SUFFIXES lib lib64 cuda/lib cuda/lib64 lib/x64)
+
+find_library(CUDA_cuda_LIBRARY cuda ${CUDA_TOOLKIT_ROOT_DIR}/lib64/stubs)
+find_library(CUDA_cudart_LIBRARY libcudart.so ${CUDA_TOOLKIT_ROOT_DIR}/lib64)
+
+target_link_libraries(${TARGET_NAME}
+    ${CUDA_cuda_LIBRARY}
+    ${CUDA_cudart_LIBRARY}
+    ${CUDA_LIBRARIES}
+    ${CUDA_CUBLAS_LIBRARIES}
+    ${CUDNN_LIBRARY})
+
+cuda_add_executable(CublasBenchmark cublas_test.cpp)   
+target_link_libraries(CublasBenchmark ${TARGET_NAME}) 
+
+install(TARGETS CublasBenchmark ${TARGET_NAME} EXPORT CublasBenchmarkTargets)
```

Comment:
Where it will be installed? Will it be put it at `SB_MICRO_PATH="${SB_MICRO_PATH:-/usr/local/bin}"`?

Reply:
not it will be installed in /usr/local/bin and /usr/local/lib, but that is what I want to ask, I want to install a bin and a .so, where I should install them, and where current superbench will find the lib from?

Reply:
All benchmark build will be called by 

> https://github.com/microsoft/superbenchmark/blob/main/superbench/benchmarks/build.sh

binary should be put at SB_MICRO_PATH, and library should be put at SB_MICRO_LIB.
You can have a try with the kernel_launch benchmark, when specify SB_MICRO_PATH, the binary will be put there.
For library, you may need to add SB_MICRO_LIB path into LD_LIBRARY_PATH just like:
> https://github.com/microsoft/superbenchmark/blob/e977bbc17f1f9da268498408617d8ecf8ad90ead/superbench/benchmarks/micro_benchmarks/micro_base.py#L148